addons/monit/monit_configurations/monit_general.tt

```diff
@@ -1,7 +1,20 @@
+set httpd port 2812 and
```

Comment:
Do we need that ?

Reply:
Yes, this is necessary to have `monit summary` working.